cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1001,12 +1001,8 @@ function Configure_Dockerd_CniNetworking {
   $cidr_range_start = Get_PodIP_Range_Start(${env:POD_CIDR})
 
   # Explanation of the CNI config values:
```

Comment:
Can you add a comment here explaining that the OutBoundNAT exceptionlist is the CIDRs for which we do not want to perform SNAT, and explain that we exclude all of the non-masquerade destination ranges listed at https://cloud.google.com/kubernetes-engine/docs/how-to/ip-masquerade-agent#default-non-masq-dests ?

Please replicate the comment for the containerd function too.

Reply:
done